pkg/apis/app/v1alpha1/prometheus.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+const (
+
+	// PrometheusDefaultPath default path
+	PrometheusDefaultPath = "/metrics"
+
+	// PrometheusDefaultScheme default scheme
+	PrometheusDefaultScheme = "http"
+)
+
+// Prometheus properties to connect with prometheus
+type Prometheus struct {
+	// Flag to allow Prometheus to scraping Kogito service.
+	Scrape bool `json:"scrape,omitempty"`
```

Comment:
When this would be `false`?

Reply:
This flag should not required. I removed it